pygmt/tests/test_figure.py

```diff
@@ -274,3 +274,47 @@ def test_figure_icc_gray():
     with pytest.warns(expected_warning=FutureWarning) as record:
         fig.psconvert(icc_gray=True, prefix="Test")
         assert len(record) == 1  # check that only one warning was raised
+
+
+def test_figure_depr_xshift():
+    """
+    Check if deprecation of xshift parameter works correctly if used.
+    """
+    fig = Figure()
+    fig.basemap(region=[0, 1, 0, 1], projection="X1c/1c", frame=True)
+    with pytest.warns(expected_warning=SyntaxWarning) as record:
+        fig.plot(x=1, y=1, style="c3c", xshift="3c")
+        assert len(record) == 1  # check that only one warning was raised
+
+
+def test_figure_depr_x():
+    """
+    Check if deprecation of X parameter works correctly if used.
+    """
+    fig = Figure()
+    fig.basemap(region=[0, 1, 0, 1], projection="X1c/1c", frame=True)
+    with pytest.warns(expected_warning=SyntaxWarning) as record:
+        fig.plot(x=1, y=1, style="c3c", X="3c")
+        assert len(record) == 1  # check that only one warning was raised
```

Comment:
Does the testing of `X` and `Y` needs to be included if `xshift` and `yshift` are already included, since we trust that the aliases translate to the appropriate arguments?

Reply:
> Does the testing of `X` and `Y` needs to be included if `xshift` and `yshift` are already included, since we trust that the aliases translate to the appropriate arguments?

I think we still need to test `X` and `Y`, because we have removed the "X=>xshift" and "Y=>yshift" aliases in this PR.